packages/nextjs/services/web3/wagmi-burner/BurnerConnector.ts

```diff
@@ -42,22 +42,19 @@ export class BurnerConnector extends Connector<StaticJsonRpcProvider, BurnerConn
 
     this.provider = new StaticJsonRpcProvider(chain.rpcUrls.default.http[0]);
     const account = await this.getAccount();
-    const chainId = await this.getChainId();
```

Comment:
Remove this call and below check. 

Reason : 
Since this function was making an RPC call `eth_getChainId` it made the exprirence of connecting to burner wallet very laggy, 

Also if you look at the function `getChainId`
```js
async getChainId(): Promise<number> {
    const network = await this.provider?.getNetwork();
    const chainId = network?.chainId ?? this.options.defaultChainId;
    if (chainId == null) {
      throw new BurnerConnectorError(BurnerConnectorErrorList.chainIdNotResolved);
    }

    return Promise.resolve(chainId);
  }
```

It's falling back to `this.options.defaultChainId` which will be our `configuredNetwork`, which is single source of truth, hence its redundant to make an RPC call with provider network to get the chainId. 


https://user-images.githubusercontent.com/80153681/227477053-be2b9688-58a3-460f-87e2-b5b594bb6985.mov